mep/people/forms.py

```diff
@@ -29,3 +30,74 @@ def __init__(self, *args, **kwargs):
         self.fields['primary_person'].queryset = \
             Person.objects.filter(id__in=person_ids)
 
+
+## SelectDisabledMixin borrowed from ppa-django
+
+
+class SelectDisabledMixin(object):
+    '''
+    Mixin for :class:`django.forms.RadioSelect` or :class:`django.forms.CheckboxSelect`
+    classes to set an option as disabled. To disable, the widget's choice
+    label option should be passed in as a dictionary with `disabled` set
+    to True::
+
+        {'label': 'option', 'disabled': True}.
+    '''
+
+    # Using a solution at https://djangosnippets.org/snippets/2453/
+    def create_option(self, name, value, label, selected, index, subindex=None,
+                      attrs=None):
+        disabled = None
+
+        if isinstance(label, dict):
+            label, disabled = label['label'], label.get('disabled', False)
+        option_dict = super().create_option(
+            name, value, label, selected, index,
+            subindex=subindex, attrs=attrs
+        )
+        if disabled:
+            option_dict['attrs'].update({'disabled': 'disabled'})
+        return option_dict
+
+
+class RadioSelectWithDisabled(SelectDisabledMixin, forms.RadioSelect):
+    '''
+    Subclass of :class:`django.forms.RadioSelect` with option to mark
+    a choice as disabled.
+    '''
+
+
+class SelectWithDisabled(SelectDisabledMixin, forms.Select):
+    '''
+    Subclass of :class:`django.forms.Select` with option to mark
+    a choice as disabled.
+    '''
+
+
+class MemberSearchForm(forms.Form):
+    '''Member search form'''
+
+    SORT_CHOICES = [
+        ('relevance', 'Relevance'),
+        ('name', 'Name A-Z'),
+    ]
+
+    query = forms.CharField(label='Keyword or Phrase', required=False,
```

Comment:
can we set arbitrary attributes on these? would be nice to be able to set `aria-label` here, since the keyword query doesn't actually have a visible `<label>` in the design.

Reply:
just kidding, I see that's exactly what you did! awesome!!